pom.xml

```diff
@@ -81,6 +83,33 @@
     </dependency>
 
     <dependency>
+      <groupId>edu.amherst.acdc</groupId>
+      <artifactId>acrepo-apple-trees</artifactId>
+      <version>${apple.trees.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.fcrepo</groupId>
```

Comment:
You should also be able to remove this exclusion -- since it's not a compile dependency in the `acrepo-apple-trees` [POM](https://repo1.maven.org/maven2/edu/amherst/acdc/acrepo-apple-trees/0.1.3/acrepo-apple-trees-0.1.3.pom).